src/main/java/edu/cornell/kfs/concur/batch/businessobject/ConcurRequestedCashAdvance.java

```diff
@@ -0,0 +1,86 @@
+package edu.cornell.kfs.concur.batch.businessobject;
+
+import java.sql.Date;
+
+import org.kuali.kfs.krad.bo.PersistableBusinessObjectBase;
+
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+
+public class ConcurRequestedCashAdvance extends PersistableBusinessObjectBase{
+    protected int concurRequestedCashAdvanceId;
```

Comment:
I am confused by this recommendation since
https://db.apache.org/ojb/docu/guides/jdbc-types.html  
indicates that the JDBC type of INTEGER should be mapped to Java type int.